PWGLF/Tasks/lambda1520_spherocity_analysis.cxx

```diff
@@ -443,17 +441,28 @@ struct lambdaAnalysis {
 
   // Processing Event Mixing
   SliceCache cache;
-  using BinningType = ColumnBinningPolicy<aod::collision::PosZ, aod::resocollision::MultV0M, aod::resocollision::Spherocity>;
-  BinningType binningPositions{{cfgVtxBins, cfgMultBins, cfgSphBins}, true};
+
+  using BinningType1 = ColumnBinningPolicy<aod::collision::PosZ, aod::resocollision::MultV0M, aod::resocollision::Spherocity>;
+  BinningType1 binningPositions1{{cfgVtxBins, cfgMultBins, cfgSphBins}, true};
+
+  using BinningType2 = ColumnBinningPolicy<aod::collision::PosZ, aod::resocollision::MultV0M>;
+  BinningType2 binningPositions2{{cfgVtxBins, cfgMultBins}, true};
 
   void processMix(resoCols& collisions, resoTracks const& tracks)
   {
 
     LOGF(debug, "Event Mixing Started");
     auto tracksTuple = std::make_tuple(tracks);
```

Comment:
```suggestion
  void processMix(resoCols& collisions, resoTracks const& tracks)
  {

    LOGF(debug, "Event Mixing Started");
    using BinningType1 = ColumnBinningPolicy<aod::collision::PosZ, aod::resocollision::MultV0M, aod::resocollision::Spherocity>;
  BinningType1 binningPositions1{{cfgVtxBins, cfgMultBins, cfgSphBins}, true};

    using BinningType2 = ColumnBinningPolicy<aod::collision::PosZ, aod::resocollision::MultV0M>;
  BinningType2 binningPositions2{{cfgVtxBins, cfgMultBins}, true};
    auto tracksTuple = std::make_tuple(tracks);
```

When you use the variable binning, the definition must be placed in the process function.